2025-06-13 20:50:24,327 [INFO] Iniciando aplicación OBD2 Scanner...
2025-06-13 20:50:24,563 [DEBUG] Using selector: _Selector
2025-06-13 20:50:24,565 [CRITICAL] Excepción no capturada:
Traceback (most recent call last):
  File "/Users/ignacioeltit/git/proyecto01junio/ob2_nuevo7junio/scanner-obd2/src/main.py", line 507, in <module>
    win = DataVisualizer(lambda: {}, pid_manager=None, elm327=None, parse_pid_response=None, backend=backend)
  File "/Users/ignacioeltit/git/proyecto01junio/ob2_nuevo7junio/scanner-obd2/src/ui/data_visualizer.py", line 110, in __init__
    self.buffer_timer.timeout.connect(self.consume_data_buffer)
AttributeError: 'DataVisualizer' object has no attribute 'consume_data_buffer'
